Index: dbus-test-runner-0.0.4/tests/Makefile.am
===================================================================
--- dbus-test-runner-0.0.4.orig/tests/Makefile.am	2011-11-28 23:19:05.000000000 +0100
+++ dbus-test-runner-0.0.4/tests/Makefile.am	2012-01-13 12:29:48.013503413 +0100
@@ -76,7 +76,7 @@
 	@chmod +x $@
 DISTCLEANFILES += testcat.output.txt testcat.output.cat1.txt testcat.output.cat2.txt testcat.output.cat1.filtered.txt testcat.output.cat2.filtered.txt
 
-TESTS += test-bustle
+#TESTS += test-bustle
 test-bustle: Makefile.am test-bustle.reference
 	@echo "#!/bin/sh -e" > $@
 	@echo $(DBUS_RUNNER) --bustle-data \"$(builddir)/test-bustle.bustle\" --task $(srcdir)/test-bustle-list.sh --ignore-return >> $@
@@ -85,7 +85,7 @@
 	@chmod +x $@
 DISTCLEANFILES += test-bustle.bustle test-bustle.filtered
 
-TESTS += test-bustle-data
+#TESTS += test-bustle-data
 test-bustle-data: Makefile.am
 	@echo "#!/bin/sh -e" > $@
 	@echo "$(DBUS_RUNNER) --bustle-data \"$(builddir)/test-bustle-data.bustle\" \\" >> $@
